service/controller/resource/heartbeatrouting/receiver/receiver.go

```diff
@@ -14,24 +15,39 @@ import (
 	"github.com/giantswarm/prometheus-meta-operator/service/key"
 )
 
-func toReceiver(cluster metav1.Object, installation string, opsgenieKey string) (alertmanagerconfig.Receiver, error) {
+func toReceiver(cfg alertmanagerconfig.Config, cluster metav1.Object, installation string, opsgenieKey string) (alertmanagerconfig.Receiver, error) {
 	u, err := url.Parse(fmt.Sprintf("https://api.opsgenie.com/v2/heartbeats/%s/ping", key.HeartbeatName(cluster, installation)))
 	if err != nil {
 		return alertmanagerconfig.Receiver{}, microerror.Mask(err)
 	}
 
+	httpConfig := &promcommonconfig.HTTPClientConfig{}
+	if cfg.Global != nil && cfg.Global.HTTPConfig != nil {
+		httpConfigCp, err := cloneHttpConfig(cfg.Global.HTTPConfig)
+		if err != nil {
+			return alertmanagerconfig.Receiver{}, err
+		}
+		httpConfig = httpConfigCp
+		// httpConfig = cfg.Global.HTTPConfig
```

Comment:
This line can be removed

Reply:
Oh yeah, sorry this was just me testing to see if the cloning works. I'll remove this.

Reply:
Done.